modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,67 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
 ----
-<1> Set the termination field to reencrypt.
-<2> Add the contents of the *_admin-ca_* file.
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
 
-. Use the following command to create the service:
+.. Run the following commnd to add the Elasticsearch CA certificate to the route YAML you created:
 +
 ----
-$ oc create -f my_es_route.yaml
+$cat ./admin-ca | sed -e "s/^/      /" >> <file-name>.yaml
+----  
+
+.. Run the following command to create the route:
++
+----
+$oc create -f <file-name>.yaml
+
+route.route.openshift.io/elasticsearch created
 ----
++
+For an example reencrypt route object, see Re-encryption Termination.
++
+//This line ^^ will be linked when the topic is available.
+
+. Check that the Elasticsearch service is exposed:
 
-. Check that the ElasticSearch service is exposed:
+.. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-curl --silent --insecure -H "Authorization: Bearer $( oc whoami -t )" "https://$( oc get route elasticsearch -o jsonpath='{.spec.host}' ):443/.operations.*/_search" | jq
+$ token=$(oc whoami -t)
 ----
+
+.. Run a command similar to the following to access Elasticsearch through the exposed route:
++
+----
+curl -tlsv1.2 -v --insecure -H "Authorization: Bearer me3IL_WmD_I_McBUm2uhxIayUKped-H3L1njlRRxPlE" "https://elasticsearch-openshift-logging.apps.ntest10.devcluster.openshift.com/.operations.*/_search?size=1" | jq
```

Comment:
Please remove the verbose option "-v" and replace the my test token and elasticsearch fqdn values with the above variables, respectively.
`$ curl -tlsv1.2 --insecure -H "Authorization: Bearer $token" "https://$elasticsearch_fqdn/.operations.*/_search?size=1" | jq`

Reply:
@nhosoi That is great! Much easier for the user!!